src/components/Volunteer/Process.js

```diff
@@ -243,17 +198,20 @@ export default function ProcessSectopm() {
         <h2 className={classes.h2Text} style={{ color: "#4529AE" }}>
           Sounds good? Send us an email.
         </h2>
+        <div
+          style={{
+            textAlign: "center",
+          }}
+        >
+          <a href="/contact/">
```

Comment:
Remove the a tag and insert the "link" prop which sends it to "/volunteer".

Reply:
I added the link and removed the a tag. But just to clarify, it should route to /contact no? Because this section is the volunteer section, so it wouldn't make sense to have this button route to /volunteer. 

I also removed the a tag and updated with the link prop for the volunteer/SkillSet.js file and WhoWeAre/TimelineSection.js file.

Reply:
Yes you are right it is the contact section